tests/Doctrine/Tests/Common/Annotations/DocParserTest.php

```diff
@@ -1537,6 +1538,55 @@ public function testWillParseAnnotationSucceededByANonImmediateDash(): void
         self::assertCount(1, $result);
         self::assertInstanceOf(SomeAnnotationClassNameWithoutConstructorAndProperties::class, $result[0]);
     }
+
+    public function testNamedArgumentsConstructorAnnotation(): void
+    {
+        $result = $this
+            ->createTestParser()
+            ->parse('/** @NamedAnnotation(bar=2222, foo="baz") */');
```

Comment:
Additionally I would also expect that another test guarantees that a different order of the parameters isn't breaking the build for this feature. `/** @NamedAnnotation(foo="baz", bar=2222) */`